wallet/wallet.go

```diff
@@ -2496,6 +2492,102 @@ func (w *Wallet) fetchTicketDetails(ns walletdb.ReadBucket, hash *chainhash.Hash
 	return ticketDetails, nil
 }
 
+// TicketSummary contains the properties to describe a ticket's current status
+type TicketSummary struct {
+	Ticket  *TransactionSummary
+	Spender *TransactionSummary
+	Status  TicketStatus
+}
+
+// TicketStatus describes the current status a ticket can be observed to be.
+type TicketStatus uint
+
+//go:generate stringer -type=TicketStatus -linecomment
+
+const (
+	// TicketStatusUnknown any ticket that its status was unable to be determined.
+	TicketStatusUnknown TicketStatus = iota // unknown
+
+	// TicketStatusUnmined any not yet mined ticket.
+	TicketStatusUnmined // unmined
+
+	// TicketStatusImmature any so to be live ticket.
+	TicketStatusImmature // immature
+
+	// TicketStatusLive any currently live ticket.
+	TicketStatusLive // live
+
+	// TicketStatusVoted any ticket that was seen to have voted.
+	TicketStatusVoted // voted
+
+	// TicketStatusMissed any ticket that has yet to be revoked, and was missed.
+	TicketStatusMissed // missed
+
+	// TicketStatusExpired any ticket that has yet to be revoked, and was expired.
+	TicketStatusExpired // expired
+
+	// TicketStatusUnspent is a matured ticket that has not been spent.  It
```

Comment:
I tried to locate this scenario in the code but I can't find it. `TicketStatusUnspent` actually seems to be unused.

Reply:
It's used now.